runtime/runc_jailer.go

```diff
@@ -412,6 +418,8 @@ func (j runcJailer) setDefaultConfigValues(cfg *Config, socketPath string, spec
 		spec.Process.Args = cmd.Args
 	}
 
+	spec.Linux.CgroupsPath = j.CgroupPath()
```

Comment:
nit: can we add a debug log saying `using cgroup path xyz` around here?

Reply:
Yep, fixed!